src/routes/docs/products/databases/bulk-operations/+page.markdoc

```diff
@@ -21,6 +21,18 @@ This atomicity ensures:
 
 For example, if you attempt to create 100 documents and one fails due to a validation error, none of the 100 documents will be created.
 
+# Plan limits {% #plan-limits %}
+
+Bulk operations have different limits based on your Appwrite plan:
+
+| Plan | Documents per request |
+|------|----------------------|
+| Free | 100 |
+| Pro  | 1,000 |
+| Scale | Customizable |
+
+These limits apply to all bulk operations including create, update, upsert, and delete operations. If you need higher limits than what the Pro plan offers, you can upgrade to the Scale plan and [contact support](https://appwrite.io/contact-us) to customize your limits based on your specific requirements.
```

Comment:
> you can upgrade to the Scale plan

This wouldn't give them a custom limit.

> [contact support](https://appwrite.io/contact-us)

This isn't the right way to contact support.

Reply:
if they want custom limits, they need to inquire about a custom plan via https://appwrite.io/contact-us/enterprise